sdk/servicebus/service-bus/test/streamingReceiver.spec.ts

```diff
@@ -1133,3 +1133,176 @@ describe("Streaming - Not receive messages after receiver is closed", function()
     await testReceiveMessages();
   });
 });
+
+describe("Streaming - onDetached", function(): void {
+  this.afterEach(async () => {
```

Comment:
Is the `this.` needed for afterEach()?

Also...I have now witnessed the fall of one of my heroes...because that looks like a UUOA. :|

Reply:
What is a UUOA?

Reply:
Useless use of await. The thing you find in every one of my PRs ;)

Reply:
The `this` shouldn't be needed, that was vscode autocompleting for me...
I did the await thing just to stay consistent with the rest of the file/project. Agreed it isn't needed, I can change that.

Reply:
Fixed